src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java

```diff
@@ -62,4 +65,20 @@ public void shouldPassValidationAndGiveSuccessWhenApplicant1HelpWithFeesApplied(
             .isEqualTo(expectedResponse(RESPONSE_HWF));
     }
 
+    @Test
+    public void shouldPassValidationAndGiveSuccessWhenJointApplicationApplied() throws IOException {
```

Comment:
this should probably be an integration test

Reply:
theres givenValidJointCaseDataThenReturnResponseWithNoErrors, can remove this if you want.

Reply:
yes please